dist/index.html

```diff
@@ -8,33 +8,56 @@
   <title>
     Rat Leader Board
   </title>
-<script defer src="bundleb1b9fb45add252067512.js"></script></head>
+  <script src="https://kit.fontawesome.com/b89b7c8adf.js" crossorigin="anonymous"></script>
+<script defer src="bundlefd9794da417207084c8a.js"></script></head>
 
 <body>
-  <main class="main-section">
-    <div class="main-section-title">
-      <h1 class="main-title-text">Leaderboard</h1>
-    </div>
-    <div class="main-content">
-      <div class="leaderboard-container">
-        <div class="leaderboard-title-container">
-          <h3 class="recent-scores-title">Recent scores</h3>
-          <button class="refresh-btn">Refresh</button>
+
+  <div class="background-container">
+    <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1231630/moon2.png" alt="">
+    <div class="stars"></div>
+    <div class="twinkling"></div>
+    <div class="clouds"></div>
+
+    <header>
+      <div class="logo" onclick="window.location.reload()">Leader Board</div>
+      <ul>
+        <li id="leaderBoard">Score board</li>
+        <div class="slash"></div>
+        <li id="addScores">Add Scores</li>
+        <div class="slash"></div>
+        <li id="contact">Contact</li>
+      </ul>
+    </header>
+    <main class="main-section">
+      <div class="main-section-title">
+        <h1 id="boardTitle" class="main-title-text">Leaderboard</h1>
+        <h1 id="addTitle"class="main-title-text hidden">Add New Score</h1>
+        <h1 id="contactTitle" class="main-title-text hidden">Our Contact</h1>
+      </div>
+      <div class="main-content">
+        <div class="leaderboard-container">
+          <div class="leaderboard-title-container">
+            <h2 class="recent-scores-title">current leaders</h2>
+            <button class="refresh-btn">Refresh</button>
+          </div>
+          <div class="leaderboard">
+            <div class="score">Name: <span>Score</span></div>
+          </div>
         </div>
```

Comment:
- [ Required ] Great job styling the UI :clap: :clap: I must admit it looks super good, however, the `leaderboard` and the `add score` form need to be on the same page as illustrated on the `project requirements` Please see the screenshots below and rectify this.

## Your implementation
![Screenshot from 2023-02-10 23-22-56](https://user-images.githubusercontent.com/16392046/218192503-07033d43-157d-49e7-afca-e0b5e70beafa.png)


## Project requirements layout
<img src="https://github.com/microverseinc/curriculum-javascript/raw/main/leaderboard/images/leaderboard_wireframe.png" />